bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/handler/DoorbellHandler.java

```diff
@@ -411,6 +429,23 @@ private void stopMotionOffJob() {
         }
     }
 
+    @SuppressWarnings({ "unchecked", "null" })
```

Comment:
Are both required ? Even "null" ?

Reply:
You are right, I can do some work on it.
Unchecked could be removed by using another method with the class as argument, and not the class name.
The null warning is not here anymore after the @Nullable annotation set on audioSinkRegistration (your next comment)

Done